conpot/conpot.cfg

```diff
@@ -39,6 +39,9 @@ host = taxiitest.mitre.org
 port = 80
 inbox_path = /services/inbox/default/
 use_https = False
+
+[stix]
+protocols = ["modbus", "snmp", "http", "s7comm"]
```

Comment:
But if I remove it, wont it be like before, hard-coded to stix_transform.py ?
Will remove all necessary files. Sorry :)